en-US/XYZ.xml

```diff
@@ -16,6 +16,41 @@
 			</listitem>
 
 		</varlistentry>
+		 <varlistentry id="x86-64">
+			<term role="true">x86_64</term>
+			 <listitem>
+				<para>
+					<emphasis>n.</emphasis> A 64-bit version of the x86 architecture. 
+					Use this term when referring to operating systems and server instances, for example Red&nbsp;Hat Enterprise Linux, Fedora, CoreOS and other Linux distributions. 
+					Use this format without backticks in general cases when referring to system architecture. 
+					Use this format in backticks when referring to architecture as a value or parameter. 
+				</para>
+				 <para>
+					Cloud providers can use different formats of this term when using architectures. 
```

Comment:
Consider: Cloud providers might use different formats of this term when discussing architectures.